external/fv3fit/fv3fit/emulation/models/_core.py

```diff
@@ -56,7 +61,25 @@ def get_combine_from_arch_key(key: str):
         key: key of core architecture block being used
     """
 
-    if key == "rnn":
+    if key == "rnn" or key == "rnn-v1":
         return CombineInputs(-1, expand_axis=-1)
     else:
         return CombineInputs(-1, expand_axis=None)
+
+
+def get_outputs_from_arch_key(key: str, output_features: Mapping[str, int]):
```

Comment:
Type hint for the output type pls
```suggestion
def get_outputs_from_arch_key(key: str, output_features: Mapping[str, int]) -> tf.keras.Layer:
```